### PR TITLE
blog(kaoto): Kaoto 2.0 TP1 Release

### PR DESCRIPTION
**Pull request by lordrip**
## Introducing Kaoto 2.0 (Tech Preview)
We are thrilled to announce the first tech preview release of Kaoto 2.0, marking a significant milestone in the evolution of the Kaoto project.

Kaoto is a no-code / low-code editor for Apache Camel integrations, which lets you create and edit your routes in a graphical way.

### CODE REVIEW

File: content/blog/2024/01/kaoto/index.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+### Kamelet Integration
+We now support adding Kamelets in a Camel Route, expanding your options, and simplifying integration tasks.
+
+### Lightweight
+This new version of Kaoto is a UI-only application, which makes it more portable for embedding it in different environments.
+
+
+## What comes next?
+
```
Comment (lhein) [inline content/blog/2024/01/kaoto/index.md:59]: remove the empty line?

File: content/blog/2024/01/kaoto/index.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+![Beans support](beans.gif)
+
+### Fresh Icons
+Check out the new set of icons which will help you to understand your integration better without looking at the details.
+
+### Dynamic Labels
+Experience dynamic labels for nodes, improving clarity and making it easier to understand and manage complex route structures. You can provide descriptions for your steps which will then show as the labels on the canvas.
+
+### “setHeaders” Processor
+We added support for the new [`setHeaders` processor](https://camel.apache.org/components/next/eips/setHeaders-eip.html), enhancing your ability to add multiple headers effortlessly.
```
Comment (apupier) [inline content/blog/2024/01/kaoto/index.md:45]: must be a relative link (there should an error at build time)

File: content/blog/2024/01/kaoto/index.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+### Copy Routes to Clipboard
+Take advantage of the "Copy Route to Clipboard" feature, perfect for leveraging the [JBang run from clipboard functionality](https://camel.apache.org/manual/camel-jbang.html#_run_from_clipboard).
```
Comment (apupier) [inline content/blog/2024/01/kaoto/index.md:48]: must be a relative link (there should an error at build time)

File: content/blog/2024/01/kaoto/index.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+[Run route from clipboard](JBangClipboard.webm)
```
Comment (apupier) [inline content/blog/2024/01/kaoto/index.md:49]: Have you tried it with this syntax? I would be surprised that it works

**Review by apupier — CHANGES_REQUESTED** [review]

File: content/blog/2024/01/kaoto/index.md (hunk 1 above)
Comment (lordrip) [reply content/blog/2024/01/kaoto/index.md]: Done

**Review by oscerd — APPROVED** [review]
Links seems to be fine, let's see if it builds.

File: content/blog/2024/01/kaoto/index.md (hunk 2 above)
Comment (lordrip) [reply content/blog/2024/01/kaoto/index.md]: Done

File: content/blog/2024/01/kaoto/index.md (hunk 4 above)
Comment (lordrip) [reply content/blog/2024/01/kaoto/index.md]: Switched to Youtube instead

File: content/blog/2024/01/kaoto/index.md (hunk 3 above)
Comment (lordrip) [reply content/blog/2024/01/kaoto/index.md]: Done

**Review by davsclaus — APPROVED** [review]

**Comment by apupier** [comment]
current build error is due to Camel Quarkus, fix in progress https://github.com/apache/camel-quarkus/pull/5683

**Comment by lordrip** [comment]
> current build error is due to Camel Quarkus, fix in progress [apache/camel-quarkus#5683](https://github.com/apache/camel-quarkus/pull/5683)

I closed my PR since I couldn't manage to build it locally, but @jamesnetherton came to the rescue to save the day: https://github.com/apache/camel-quarkus/pull/5684 :muscle:

**Review by apupier — APPROVED** [review]

**Pull request merged**